MaterialComponents.podspec

```diff
@@ -1906,6 +1906,25 @@ Pod::Spec.new do |mdc|
     end
   end
 
+  mdc.subspec "TextControls+Theming" do |extension|
+    extension.ios.deployment_target = '9.0'
+    extension.public_header_files = "components/#{extension.base_name.split('+')[0]}/src/#{extension.base_name.split('+')[1]}/*.h"
+    extension.source_files = [
+      "components/#{extension.base_name.split('+')[0]}/src/#{extension.base_name.split('+')[1]}/*.{h,m}",
+      "components/#{extension.base_name.split('+')[0]}/src/#{extension.base_name.split('+')[1]}/private/*.{h,m}"
+    ]
+    extension.dependency "MaterialComponents/#{extension.base_name.split('+')[0]}"
+    extension.dependency "MaterialComponents/Typography"
```

Comment:
Shouldn't this also include `schemes/Typography` and `schemes/Color`?

Reply:
Done.